pkg/database/db.go

```diff
@@ -33,6 +33,11 @@ type DB struct {
 	*sql.DB
 }
 
+// ConnectionString returns the connection string in driver_type://data_source_name format
```

Comment:
This function already exists, named URL()

Reply:
Thanks for pointing it out!